habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+import shutil
+import sys
+import traceback
+import zipfile
+
+import git
+
+repo = git.Repo(".", search_parent_directories=True)
+dir_path = repo.working_tree_dir
+# Root data directory. Optionaly overridden by input argument "--data_path".
+data_path = os.path.join(dir_path, "data")
+
+data_sources = {}
+data_groups = {}
+
+
+def initialize_test_data_sources(data_path):
+    global data_sources
+    global data_groups
+    r"""Initializes data_sources and data_groups dicts with a variable data_path path.
+    """
+    # dict keyed by uids with download specs for various individual test assets and datasets
+    data_sources = {
+        # "example_uid": {
+        #   "source": the URL download link
+        #   "package_name": the filename of the downloaded compressed package
+        #   "download_pre_args": (optional)(wget) commands preceding filename
+        #   "download_post_args": (optional)(wget) commands follow filename
+        #   "unpack_to": destination path for uncompression of the download package
+        #   "link": symlink to the data directory pointing to the active version directory
+        #   "version": data version tag
+        # }
+        "habitat_test_scenes": {
+            "source": "http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip",
+            "package_name": "habitat-test-scenes.zip",
+            "unpack_to": data_path + "../",
```

Comment:
Doesn't this break if the user overrides the `data_path`? Since that particular zip file hard-codes `data`.

I would be a fan of changing that zip file to not contain `data/scene_datasets/habitat-test-scenes`. These dataset zip files shouldn't be making assumptions about where they might live in the larger project. And then maybe this whole idea of separate `unpack_to` and `link` paths could go away (there'd just be `link`).